test/Microsoft.AspNet.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -15,79 +14,336 @@ public class OptionTagHelperTest
     {
         // Original content, selected attribute, value attribute, selected values (to place in FormContext.FormData)
         // and expected output (concatenation of TagHelperOutput generations).
```

Comment:
still incorrect.  the last value is the expected `TagHelperOutput`, not a "concatenation of TagHelperOutput generations".